### ISSUE TITLE
[image_picker]IOS 4/3 images not working with ImagePicker in VS Code

### ISSUE THREAD

**Issue by jerryhamby** [flutter/flutter#44310]
Note I'm testing with 2 devices: (not testing using camera mode yet)

1- iPhone 7 (IOS 13.2) (has 4/3 and square)
2- iPhone 11 (IOS 13.2) (has 4/3, 16/9 and square)
Using ImagePicker (image_picker: ^0.6.1+10), (Flutter 1.9.1+hotfix.5)

I have found that when I select an image from the library that was taken as follows, something is wrong:
1- portrait mode using 4/3 setting (not landscape) the device is returning the width is
    greater than the height. (this is opposite of what I want or expect)

Something is either wrong with the IOS device or ImagePicker is returning the wrong info.
Need some help or advice.

**Comment by iapicca** [comment]
Hi @jerryhamby 
this issue could have been fixed very recently,
please let me know if the proposed [solution ](https://github.com/flutter/flutter/issues/39311#issuecomment-550499632) worked for you
thank you

**Comment by jerryhamby** [comment]
Hi Francecsco,

I don’t see any solution that I haven’t already tried. I’m using the latest version of ImagePicker (0.6.1+10 <https://pub.dev/packages/image_picker/versions/0.6.1+10>).

It’s still returning the width is greater than the height, which is false info. I took the image to be in portrait mode, not landscape.

thumbnail = await ImagePicker.pickImage(
      source: ImageSource.gallery,
      maxWidth: 540.0
    );



Jerry

**Comment by cyanglaz** [comment]
> 1- portrait mode using 4/3 setting (not landscape) the device is returning the width is
> greater than the height.

@jerryhamby Could you elaborate what did you mean by this? Did you see the picked image has a wrong presentation, or is the meta data of the image wrong?

**Comment by jerryhamby** [comment]
Here are the steps I used to test this problem:

1- Use iPhones to capture an image, I used the portrait mode to capture that image. It looks normal if I use to phone to view that image (the image is not squashed).

2- I open my Flutter app to grab that image, expecting it to return the image in portrait mode, not landscape.

3- I expect the height to be larger than the width. (It’s not)

4- The meta data says the image is wider not taller (this is incorrect).

5- So my code looks at the metadata, sees that it’s a landscape image, and display everything in landscape, which in turn will squash my portrait image.

So the wrong metadata is causing my code to the wrong thing with the image (presentation is wrong).

Hope this is clear, this imaging stuff is very important to my app. Basically my app allows the user to photograph their collection of trading cards and store everything is Firestore Storage.

It’s hard for me to test in simulator mode but the device will screw up every time.

Jerry

**Comment by cyanglaz** [comment]
@jerryhamby Thanks for providing those steps!
Could you also provide a reduced test case, (An example app with minimum code) so I can reproduce the issue and further investigate?

**Comment by jerryhamby** [comment]
@cyanglaz Per your use-case request, I started a new Imagepicker example from scratch. I may have found the problem and today it's working as expected.

(image: ^2.1.8) was the problem (for reducing the file size, also for getting width and height)
I'm now exclusively using flutter_native_image: ^0.0.4 and (so far) my images are looking correct. If I experience further problems I will post, keeping my fingers crossed. Tested on both simulator and device.
